sonic-net/sonic-buildimage
# [Supermicro]: Add a new supported device and platform, SSE-T7132S.

#### Why I did it
* platform/innoviunm: Add a new supported device and platform, SSE-T7132S

* Switch Vendor: Supermicro
* Switch SKU:  Supermicro_sse_t7132s
* ASIC Vendor: innovium
* Swich ASIC: TL7
* Port Configuration: 32x400G

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
1. Add the files into device/supermicro/x86_64-supermicro_sse_t7132s-r0/ and platform/innovium/sonic-platform-modules-supermicro/ for SSE-T7132S.
2. Modify the make files in platform/innovium/ for SSE-T7132S.
#### How to verify it
1. Build a Innovium target for SSE-T7132S.

2. Run this image on SSE-T7132S and verify all dockers are up and test basic commands like:
show version
show platform summary
show platform syseeprom
show platform fan
show platform psustatus
show platform firmware status
show platform temperature
show platform ssdhealth

3. Verify ports are up using "show interface status" command



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
